src/__test__/integration/sigService.spec.ts

```diff
@@ -34,7 +34,7 @@ describe('Sig service test suite', () => {
         const result = await callSig(peer, 'CustomSig');
 
         expect(result.success).toBe(true);
-        const isSigCorrect = await customSig.verify(result.signature, data);
+        const isSigCorrect = await customSig.verify(result.signature as number[], data);
```

Comment:
Why not just `result.signature!` ?

Reply:
this is needed to satisfy typescript. Because it's a test  and we are sure it's not null - it's ok to just use "as" keyword. In other cases compiler will complain now so we have something to think about and maybe check explicitly that it's not null if needed

Reply:
I mean use bang symbol

```
 const isSigCorrect = await customSig.verify(result.signature!, data);
```